### PR TITLE
Add AMD-newlib license

### LINKED ISSUE #2395: New license request: AMD-Permissive [SPDX-Online-Tools]
**1.** License Name: AMD Permissive License
**2.** Short identifier: AMD-Permissive
**3.** License Author or steward: Unknown
**4.** Comments: This license is used in newlib sources and was found during SPDX review for the gcc package:

https://gitlab.com/fedora/legal/fedora-license-data/-/issues/480
**5.** License Request Url: http://tools.spdx.org/app/license_requests/351
**6.** URL(s): https://sourceware.org/git/?p=newlib-cygwin.git;a=blob;f=newlib/libc/sys/a29khif/_close.S;h=04f52ae00de1dafbd9055ad8d73c5c697a3aae7f;hb=HEAD
**7.** OSI Status: Unknown
**8.** Example Projects: https://sourceware.org/git/?p=newlib-cygwin

### PR DESCRIPTION
**Pull request by siddhesh**
Resolves #2395.

### CODE REVIEW

File: src/AMD-newlib.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="AMD-newlib"
+   name="AMD newlib License" listVersionAdded="">
```
Comment (xsuchy) [inline src/AMD-newlib.xml:4]: Can you add to listVersionAdded=3.24, please?

**Comment by xsuchy** [comment]
Otherwise it LTGM.

File: src/AMD-newlib.xml (hunk 1 above)
Comment (siddhesh) [reply src/AMD-newlib.xml]: Ack, done.

**Review by xsuchy — APPROVED** [review]
Thank you.

**Pull request merged**